server/api/event/index.js

```diff
@@ -1,12 +1,17 @@
 var router = require('express').Router();
 var db = require('../../db');
 
-// Return a list of all events
+// Return a list of all active events
 router.get('/', function(req, res) {
- db.Event.findAll()
-   .then(function(results){
-     res.json(results);
-   });
+ db.Event.findAll({
+  where: {
+    end: {
+      $gt: new Date() // Filters events where end date is greater than the current timestamp
```

Comment:
try using new Date(new Date().getTime() + 60 * 60 * 24 * 1000)

when we save to the database, we use this format